docs/usage/bot-comparison.md

```diff
@@ -0,0 +1,78 @@
+# Bot comparison
+
+This page explains the key differences between Renovate and Dependabot.
+
+## Table
+
+| Name of feature              | Dependabot                                                                                                                                                                   | Renovate                                                           |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------ |
+| Dependency Dashboard         | No                                                                                                                                                                           | Yes                                                                |
+| Grouped updates              | Yes, create [`groups`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups) manually     | Yes, use community-provided groups, or create your own             |
+| Monorepo support             | unknown                                                                                                                                                                      | Yes                                                                |
```

Comment:
I don't know if Dependabot supports monorepos. The closest information I could find is [GitHub Docs, troubleshooting Dependabot errors](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/troubleshooting-dependabot-errors#dependabot-timed-out-during-its-update):

> ## Dependabot timed out during its update
>
> Dependabot took longer than the maximum time allowed to assess the update required and prepare a pull request. This error is usually seen only for large repositories with many manifest files, for example, npm or yarn monorepo projects with hundreds of package.json files. Updates to the Composer ecosystem also take longer to assess and may time out.